{"id": 592879, "name": "Shahsank J Shetty", "username": "22f3001011", "avatar_template": "/user_avatar/discourse.onlinedegree.iitm.ac.in/22f3001011/{size}/66771_2.png", "created_at": "2025-02-09T15:46:42.193Z", "cooked": "<p>For the 8th question, i am getting an error that tells me i have not run the action, though i manually triggered it and confirmed the commit was made. Cant figure out whats wrong.</p>", "post_number": 279, "post_type": 1, "posts_count": 370, "updated_at": "2025-02-09T15:46:42.193Z", "reply_count": 1, "reply_to_post_number": null, "quote_count": 0, "incoming_link_count": 0, "reads": 58, "readers_count": 57, "score": 16.2, "yours": false, "topic_id": 165959, "topic_slug": "ga4-data-sourcing-discussion-thread-tds-jan-2025", "display_username": "Shahsank J Shetty", "primary_group_name": "ds-students", "flair_name": "ds-students", "flair_url": null, "flair_bg_color": "", "flair_color": "", "flair_group_id": 294, "badges_granted": [], "version": 1, "can_edit": false, "can_delete": false, "can_recover": false, "can_see_hidden_post": false, "can_wiki": false, "read": true, "user_title": null, "bookmarked": false, "actions_summary": [{"id": 2, "can_act": true}, {"id": 6, "can_act": true}, {"id": 3, "can_act": true}, {"id": 4, "can_act": true}, {"id": 8, "can_act": true}, {"id": 10, "can_act": true}, {"id": 7, "can_act": true}], "moderator": false, "admin": false, "staff": false, "user_id": 17948, "hidden": false, "trust_level": 1, "deleted_at": null, "user_deleted": false, "edit_reason": null, "can_view_edit_history": true, "wiki": false, "post_url": "/t/ga4-data-sourcing-discussion-thread-tds-jan-2025/165959/279", "user_cakedate": "2023-06-03", "reactions": [], "current_user_reaction": null, "reaction_users_count": 0, "current_user_used_main_reaction": false, "can_accept_answer": false, "can_unaccept_answer": false, "accepted_answer": false, "topic_accepted_answer": null}